### PR TITLE
Update: Shivam-katare.json

### PR DESCRIPTION
**Pull request by Shivam-Katare**
- Updated my Linkfree profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5275"></a>

### CODE REVIEW

**Review by iamthecloverly — DISMISSED** [review]

**Review by TRohit20 — DISMISSED** [review]

**Review by mahfuztbt — APPROVED** [review]

**Review by sidharthviz — DISMISSED** [review]

**Review by amandamartin-dev — APPROVED** [review]

File: data/Shivam-Katare.json [hunk 1]
```diff
@@ -1,33 +1,118 @@
...
+     "title": "First Job",
+     "date": "July 2022",
+     "icon": "FaBriefcase",
+     "color": "blue",
+     "description": "I got the job as a Frontend(React JS) Developer at a Cloud Security based company."
+   },
+   {
+     "title": "First Open Source Community",
+     "date": "April 2022",
+     "icon": "FaHandshakeO",
```
Comment (kumarsonsoff3) [inline data/Shivam-Katare.json:112]: You probably be using an invalid icon here. It should be `FaHandshake`

```suggestion
     "icon": "FaHandshake",
```

Comment (Shivam-Katare) [reply data/Shivam-Katare.json]: @kumarsonsoff3 fixed the icon name.

File: data/Shivam-Katare.json [hunk 2]
```diff
@@ -1,33 +1,118 @@
...
+  "bio": "Fullstack Development | Contributer at **[ReactPlay](https://reactplay.io/)** | Technical Content Writer | DevOps Enthusiast",
+
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "ReactJS",
+    "Blogging",
+    "DevOps"
+  ],
+
```
Comment (schmelto) [inline data/Shivam-Katare.json:13]:
```suggestion
```

Comment (Shivam-Katare) [reply data/Shivam-Katare.json]: @schmelto it's done.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**